scripts/manager_action_damage_4e_tmt.lua

```diff
@@ -0,0 +1,90 @@
+-- 
+-- Please see the license.txt file included with this distribution for 
+-- attribution and copyright information.
+--
+
+rApplyHpCombatantAction = nil;
+
+function onInit()
+    initializeActions();
+end
+
+function initializeActions()
+    -- TODO: Add more parameters to this action
```

Comment:
In either the next or second update I will be adding support for using/defining 5E power actions directly in triggers. So linking to a heal action on a character sheet or making one directly in the trigger using the same UI (or *maybe* a cousin version as I did for KNK, depending on how much the ruleset's tight coupling gets in the way).

If it is easy to do for 4E, I might deprecate this in favor of that, though I have no real familiarity with the ruleset so I may not take that on.

Reply:
Healing is pretty different in 4e, so I'd have to see how it's implemented, but after that I can port it back to 4e and replace this if it's necessary.